### ISSUE TITLE
Depends on unmaintained SimpleMDE

### ISSUE THREAD

**Issue by sedubois** [zooppa/administrate-field-simple_markdown#9]
- I would recommend fixing the hyperlink to SimpleMDE in the README, which currently points to a broken repo. It looks like the proper place is https://github.com/sparksuite/simplemde-markdown-editor (see discussion https://github.com/nextstepwebs/simplemde-markdown-editor/issues/1#issuecomment-375843973).
- SimpleMDE is unmaintained since 4 years, maybe can be replaced by the fork EasyMDE https://github.com/Ionaru/easy-markdown-editor? In the future there might also be markdown support in [Trix](https://github.com/basecamp/trix/pull/737).

**Comment by michelegera** [comment]
Thanks for the heads up, I just fixed the link as suggested.

I am working on replacing SimpleMDE with EasyMDE, I should have a PR ready in a few days.

**Comment by michelegera** [comment]
Porting to EasyMDE is almost done, I am trying to troubleshoot an issue with FontAwesome icons not displaying.

**Comment by michelegera** [comment]
Actually, I figured it out 😄 

I just pushed a branch with the porting, if you want you can try it by fetching the plugin directly from this repo:

```ruby
# Gemfile

# Make sure you have the following line at the top of your Gemfile
# git_source(:github) { |repo| "https://github.com/#{repo}.git" }

gem 'administrate-field-simple_markdown', github: 'zooppa/administrate-field-simple_markdown', branch: 'switch-to-easymde'
```

**Comment by sedubois** [comment]
Thanks @michelegera, I gave the branch a try by loading one of my `Post::Translation`s which have a `content` field of type `Field::SimpleMarkdown`, but the editor doesn't load it's just a standard textarea. The code is like this:

<img width="612" alt="Screenshot 2020-03-24 at 14 10 35" src="https://user-images.githubusercontent.com/4217871/77429059-5fc07280-6dd9-11ea-8a89-dc6f19c4573c.png">

Can it still have anything to do with the fact that the model is namespaced?

**Comment by michelegera** [comment]
Closed via 3f00c2d1b2c8c3d25ac69772aa4b33ce2108b09c
